app/static/js/userProfile.js

```diff
@@ -108,6 +114,9 @@ $(document).ready(function(){
       data: data,
       success: function(s){
           displayMessage("Saved!", "success")
+          var date = new Date(`${data.bgDate}`).toLocaleDateString()
```

Comment:
There's no reason to put your string variable into a string to pass it to `Date()`. Just use the variable.